test/integration/apiserver/apply/apply_test.go

```diff
@@ -404,9 +404,6 @@ func TestApplyGroupsManySeparateUpdates(t *testing.T) {
 	for i := 0; i < 20; i++ {
 		unique := fmt.Sprintf("updater%v", i)
 		version := "v1"
-		if i%2 == 0 {
-			version = "v1beta1"
```

Comment:
If you don't mind, maybe remove the "version" variable, and just hardcode it in the path below?

Reply:
> If you don't mind, maybe remove the "version" variable, and just hardcode it in the path below?

sure.  Done.